myuw/dao/term.py

```diff
@@ -148,6 +148,13 @@ def is_past(term, request):
     return term.get_eod_last_final_exam() < get_comparison_datetime(request)
 
 
+def is_future(term, request):
+    """
+    return true if the given term is in the future
+    """
+    return term.get_bod_first_day() > get_comparison_datetime(request)
```

Comment:
Despite that, summer A switches to current term in the front-end on the 24th of june, which is the first day of the term. Looks like instructor and student are doing the same thing, so i'm switching over to approve.